### DISCUSSION TITLE
How to open url link in default browser?

### DISCUSSION THREAD

**Discussion by ramilrgk** [termux/termux-app#3525] [Q&A] [answered]
Hello, how can I open url links in the terminal in the default browser for the system when I click on it? I don't need the built-in Termius browser. I couldn't find an option to turn this off in the settings. This is very strange behavior. I don't want to copy and paste links into my browser every time.

**Comment by twaik** [comment]
```
termux-open-url https://github.com
```

**Comment by sylirre** [comment, marked as answer]
This feature is disabled by default.

As said in changelog for v0.118, you need to edit `~/.termux/termux.properties` file:
> Allow users to directly open URL links in terminal transcript when clicked or tapped. The user can add `terminal-onclick-url-open=true` entry to `termux.properties` file to enable opening of URL links in terminal transcript when clicked or tapped. The default value is `false`. Running `termux-reload-settings` command will also update the behaviour instantaneously if changed. Implemented by `@trygveaa` in https://github.com/termux/termux-app/pull/2146. ([af16e79b](https://github.com/termux/termux-app/commit/af16e79b), [1a5a66d0](https://github.com/termux/termux-app/commit/1a5a66d0))
